File: bootstrap/v0.0.7.6/benchmarks/python/factorial.py
def factorial(n):
    if n <= 1:
        return 1
    else:
        return n * factorial(n - 1)

if __name__ == "__main__":
    sum_val = 0
    i = 1
    while i <= 20:
        result = factorial(i)
        sum_val = sum_val + result
        i = i + 1
    exit(0)

File: bootstrap/archived_runa_0.0.7.5+/v0.0.8.2/benchmarks/c/factorial.c
#include <stdio.h>

long factorial(int n) {
    if (n <= 1) {
        return 1;
    } else {
        return n * factorial(n - 1);
    }
}

int main() {
    long sum = 0;
    int i = 1;
    while (i <= 20) {
        long result = factorial(i);
        sum = sum + result;
        i = i + 1;
    }
    return 0;
}